client/src/pages/private/CohortTable.js

```diff
@@ -65,7 +65,8 @@ export default ({ cohorts }) => {
               );
             if (columnId === 'start_date' || columnId === 'end_date') {
               const d = new Date(row[columnId]);
```

Comment:
A little descriptive naming like `cloumnDate` or use
```js
return (new Date(row[columnId])).toUTCString.split(' ').slice(1, 4).join(' ');
```
+ 
Use dayjs / moment to format

Reply:
I'll use the one-line return function. What's your reasoning for why I would use a package instead?